### PR TITLE
fix rviz settings

### PR DESCRIPTION
**Pull request by tkimura4**
- Change the sytle of pointcloud (Points -> Squares).
- Fix topic name of checkpoint etc.
- Enable visualization of predicted paths of dynamic objects.

### CODE REVIEW

**Comment by wep21** [comment]
image

File: autoware_launch/rviz/autoware.rviz [hunk 1]
```diff
@@ -697,7 +697,7 @@ Visualization Manager:
               Name: DynamicObjects
               Namespaces:
                 label: true
-                path: false
+                path: true
```
Comment (wep21) [inline autoware_launch/rviz/autoware.rviz:700]: Path flag is false for detection.
https://github.com/tier4/autoware_launcher/blob/cdc3e2de5882445ad8d468a1788c95ed508a71c0/autoware_launch/rviz/autoware.rviz#L616-L621

File: autoware_launch/rviz/autoware.rviz [hunk 2]
```diff
@@ -718,7 +718,7 @@ Visualization Manager:
...
```
Comment (wep21) [inline autoware_launch/rviz/autoware.rviz:721]: Path flag is false for tracking.
https://github.com/tier4/autoware_launcher/blob/cdc3e2de5882445ad8d468a1788c95ed508a71c0/autoware_launch/rviz/autoware.rviz#L616-L621

File: autoware_launch/rviz/autoware.rviz [hunk 3]
```diff
@@ -739,7 +739,7 @@ Visualization Manager:
...
                 path confidence: true
                 position covariance: true
```
Comment (wep21) [inline autoware_launch/rviz/autoware.rviz:744]: Position covariance flag is false for prediction.
https://github.com/tier4/autoware_launcher/blob/cdc3e2de5882445ad8d468a1788c95ed508a71c0/autoware_launch/rviz/autoware.rviz#L633-L638

**Comment by wep21** [comment]
image

**Comment by wep21** [comment]
image

**Comment by wep21** [comment]
image

File: autoware_launch/rviz/autoware.rviz (hunk 1 above)
Comment (tkimura4) [reply autoware_launch/rviz/autoware.rviz]: Fixed at 0da8afb .

File: autoware_launch/rviz/autoware.rviz (hunk 2 above)
Comment (tkimura4) [reply autoware_launch/rviz/autoware.rviz]: Fixed at 0da8afb .

File: autoware_launch/rviz/autoware.rviz (hunk 3 above)
Comment (tkimura4) [reply autoware_launch/rviz/autoware.rviz]: Fixed at 0da8afb .

**Review by wep21 — APPROVED** [review]
LGTM
image

**Pull request merged**